pgctl/fuser.py

```diff
@@ -22,7 +26,7 @@ def stat(path):
         trace('fuser suppressed: %s', error)
         return None
     else:
-        return (path.st_ino, path.st_dev)
+        return StatTuple(path.st_ino, path.st_dev, path.st_nlink)
```

Comment:
Ahh gotcha. For some reason I thought the whole stat object was returned in the old version.